actool/manifest.go

```diff
@@ -250,6 +264,31 @@ func patchManifest(im *schema.ImageManifest) error {
 		}
 	}
 
+	if patchSeccompMode != "" {
+		i := 0
```

Comment:
Correct, I made it explicit in the two seccomp isolators, but behavior for others is not specified. At which level do you want to split this out? Something like introducing a `ReplaceIsolatorsByName(Isolator, []string)`?